src/com/jester/view/View.java

```diff
@@ -325,20 +385,33 @@ public void actionPerformed(ActionEvent e) {
         ImageIcon scaledSB = new ImageIcon(SBscale);
         ImageIcon scaledPB = new ImageIcon(PBscale);
         playButtonContainer.setIcon(scaledPB);
+        
+        // TODO: Play and Stop music button
         playButtonContainer.addMouseListener(new MouseAdapter() {
             private boolean playing = false;

```

Comment:
This one is causing some problems due to it being only assigned upon Mouse event notification